src/global.cpp

```diff
@@ -884,40 +890,33 @@ void fa_write (int j, str outdir, const MTL & M, const Plates & P, const FP & pp
                 if(g>0){
                     target_id[i] = M[g].id;
                 }else{
-                    if(pp[fib].broken){
-                        target_id[i] = -2;
-                    }else if(pp[fib].stuck){
-                        target_id[i] = -3;
-                    }else{
-                        target_id[i]=-1;
-                    }
+                    target_id[i]=-1;
                 }
 
                 if (g < 0) {
                     if(pp[fib].stuck){
-                        ra[i] = qNan;
-                        dec[i] = qNan;
+                        fibermask[i] = 2;
```

Comment:
Please move the fibermask magic numbers 0/2/4 into constants defined in a header file to make it easier to update them later if needed.

Reply:
Done.